### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by redwing555**
### 🎥  [Video Recording](https://drive.google.com/file/d/1sKtLAJ6SPKKJRUtk6JzoILhwDtjZwYul/view?usp=sharing)

This Pr consists of the following pages added to this project : 

- Splash screen

  - A simple page with the name of the app, and links to the sign-up and log-in pages.
  - The user should be able to register in the app with their full name, email, and password.
  - The user can log into the app using email and password.
  - If the user is not logged in, they can't access pages that require the user to be logged in.

- Home page
  - When the user logs in, they are presented with the categories page.
  - For each category, the user can see their name, icon, and the total amount of all the transactions that belong to that category.
  - When the user clicks (or taps) on a category item, the application navigates to the transactions page for that category.
  - There is a button "add a new category" at the bottom that brings the user to the page to create a new category.

- Transactions page

  - For a given category, the list of transactions is presented, ordered by the most recent.
  - At the top of the page, the user could see the total amount for the category (sum of all of the amounts of the transactions in that category).
  - There is a button "add a new transaction" at the bottom that brings the user to the page to create a new transaction.
  - When the user clicks on the "Back" button (<), the user navigates to the home page.

- New category page

  -    The user fills out a form to create a new category, indicating their name and icon.
  -    The user clicks (or taps) the "Save" button to create the new category, and is taken to the home page on success.
  -    When the user clicks on the "Back" button (<), the user navigates to the home page.

- New transaction page

  - The user fills out a form to create a new transaction with:
  - name 
  - amount 
  - The user clicks (or taps) the "Save" button to create the new category, and is taken to the transactions page for that category.
  - When the user clicks on the "Back" button (<), the user navigates to the transactions page for that category.

- Testing 
  - Unit and integration tests for all the most important components of the app.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,73 +1,70 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# Budget App
 
+An app where you can manage your budget: you can manage a list of transactions associated with a certain category, keeping a detailed record of expenses.
```
Comment (emmanuellekamwa) [inline README.md:6]: - [x] You have to deploy the app on either Heroku, Netlify, or Github pages and add the link in the readme file. Please try and implement this change.

**Review by emmanuellekamwa — CHANGES_REQUESTED** [review]
**STATUS: CHANGES REQUESTED ⛔❌**
Hello @redwing555 👋
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (peterrobert) [reply README.md:6]: This change has been made well

Comment (redwing555) [reply README.md:6]: Hello @emmanuellekamwa, 

Thanks for the review and the suggestion, I was able to deploy the app into Heroku, and add [ the link](https://budgetvio-app.herokuapp.com/)  to the readme file as suggested! 

cheers , 🥂

**Review by peterrobert — APPROVED** [review]
Hi @redwing555, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by redwing555** [comment]
Hello @peterrobert, thanks for the review have a nice weekend !

**Pull request merged**